### DISCUSSION TITLE
Meta 調整發展計畫，丹麥資料中心續建第 3 座臨時喊卡

### DISCUSSION THREAD

**Discussion by Ben6uo** [1101-GS3073-C/Python-Community-GS3073-C#551] [General]
感覺是個酷酷的東西
https://technews.tw/2022/12/23/meta-stops-planned-data-center-expansion-in-denmark/

**Comment by Yilin20021016** [comment]
剛剛就有看到 很好奇這個是什麼東西

**Comment by brian431** [comment]
Meta最近的發展真的非常坎坷，不知道他們花大筆大筆的錢到元宇宙中究竟會不會得到回報。

**Comment by williamyen0623** [comment]
我相信Meta可以讓我們進入像刀劍神域一樣Link Start的夢幻時代

**Comment by HsuChen88** [comment]
原來為了處理 AI 人工智慧運算負載，得興建好多資料中心。
